docs/faq.md

```diff
@@ -9,16 +9,16 @@ need to make a change to your build steps, you are forced to make that change ac
 your similar projects. Using a preset rather than a boilerplate keeps this process DRY.
 
 Tools like [Create React App](https://github.com/facebook/create-react-app) have
-been fantastic improvements to the tooling ecosystem, but unfortunately only work on specific
-environments like React, and do not allow simple extensibility of the build configuration. To
+been fantastic improvements to the tooling ecosystem, but unfortunately, only work on specific
+environments like React, and do not allow simple extensibility of the build configuration. To 
```

Comment:
Trailing whitespace was added here?

```suggestion
environments like React, and do not allow simple extensibility of the build configuration. To
```

Reply:
While editing it might have got added I didn't observe 😅